cypress/integration/backend-admin-ui/test_2_dataset.js

```diff
@@ -61,39 +61,85 @@ context("Admin UI Single Dataset", () => {
         });
     });
 
-    describe("Can visit tableinfo page", () => {
-      before(() => { visitAdminUI(`datasets/${testDSID}/tables/${testDSID}.table`); });
-
-      it("Can use page components", () => {
-        cy.contains('Table table');
-        cy.get('[data-test-id="accordion-Tags"]').click();
-        cy.get('[data-test-id="accordion-Concepts"]').click();
-        cy.get('[data-test-id="accordion-Columns"]').click();
-      });
+    describe("Can visit table info page", () => {
+        before(() => { visitAdminUI(`datasets/${testDSID}/tables/${testDSID}.table`); });
+
+        it("Can use page components", () => {
+            cy.contains('Table table');
+            cy.get('[data-test-id="accordion-Tags"]').click();
+            cy.get('[data-test-id="accordion-Concepts"]').click();
+            cy.get('[data-test-id="accordion-Columns"]').click();
+        });
+    });
+
+    describe("Can visit concept info page", () => {
+        before(() => { visitAdminUI(`datasets/${testDSID}/concepts/${testDSID}.concept1`); });
+
+        it("Can use page components", () => {
+            cy.contains('Concept Concept1');
+            cy.get('[data-test-id="accordion-Selects"]').first().click();
+            cy.get('[data-test-id="accordion-Connectors"]').click();
+        });
+    });
+
+    describe("Can visit connector info page", () => {
+
+        it("Can use page components", () => {
+            visitAdminUI(`datasets/${testDSID}/concepts/${testDSID}.concept1`);
+            cy.get('[data-test-id="accordion-Connectors"]').click();
+            cy.get('[data-test-id="accordion-Connectors"]').get('tr > :nth-child(1) > a').click();
+
+            cy.location('pathname').should('equal', `/admin-ui/datasets/${testDSID}/connectors/${testDSID}.concept1.column`);
+        });
+
+        it("Counts are right", () => {
+            visitAdminUI(`datasets/${testDSID}/connectors/${testDSID}.concept1.column`);
+            cy.get('[data-test-id="accordion-Filters"] > .card-header').contains("20 entries")
+            cy.get('[data-test-id="accordion-Selects"] > .card-header').contains("16 entries")
+        })
     });
 
-    describe("Can visit conceptinfo page", () => {
-      before(() => { visitAdminUI(`datasets/${testDSID}/concepts/${testDSID}.concept1`); });
+    describe("Connector page links work", () => {
+        beforeEach(() => { visitAdminUI(`datasets/${testDSID}/connectors/${testDSID}.concept1.column`); });
+
+        it("Goto datasets", () => {
+            cy.get('.breadcrumb > :nth-child(1) > a').click()
+            cy.location('pathname').should('equal', `/admin-ui/datasets`);
```

Comment:
@FabianBinder21 I used this [pattern](https://github.com/cypress-io/cypress/issues/1805#issuecomment-735796194) to ensure the page was loaded. This might avoid the wait in other places